src/MongoDB/Exception/RuntimeException.c

```diff
@@ -43,6 +119,8 @@ void php_phongo_runtimeexception_init_ce(INIT_FUNC_ARGS) /* {{{ */
 	php_phongo_runtimeexception_ce = zend_register_internal_class_ex(&ce, spl_ce_RuntimeException, NULL TSRMLS_CC);
 #endif
 	zend_class_implements(php_phongo_runtimeexception_ce TSRMLS_CC, 1, php_phongo_exception_ce);
+
+	zend_declare_property_null(php_phongo_runtimeexception_ce, ZEND_STRL("errorLabels"), ZEND_ACC_PROTECTED TSRMLS_CC);
```

Comment:
Is there a particular reason this can't be private? `hasErrorLabel()` is `final` on this class and should be the only method that accesses it.

I realize `WriteException::$writeResult` is also protected (and only accessed by `getWriteResult()`. Perhaps `protected` is needed so that we can update the property on any class? If so, I wonder if that's simply a problem with `phongo_add_exception_prop()` not using the base class entry when it calls `zend_update_property()`.

Reply:
| I realize WriteException::$writeResult is also protected (and only accessed by getWriteResult(). Perhaps protected is needed so that we can update the property on any class?

Yes, that's the case:
```
001+ Fatal error: Uncaught ReflectionException: Property errorLabels does not exist in /home/derick/dev/php/derickr-mongo-php-driver/tests/exception/bulkwriteexception-haserrorlabel-001.php
```

|  If so, I wonder if that's simply a problem with phongo_add_exception_prop() not using the base class entry when it calls zend_update_property().

It doesn't matter when I change it to use the base class (php_phongo_runtimeexception_ce). So, I think we're stuck with this.